### PR TITLE
Awesome book website with navigation 

### PR DESCRIPTION
**Pull request by AbhishekkM05**
The Third Partner @bertinKambale is no longer with us in group kindly review the project.
- Implemented Navigation for the website ✔️ 
- Added CSS for all Navigation elements. 📜 
- Implemented Javascript function to hide and display the sections. 👍 
- Fixed bugs in the project. 🐛

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -5,19 +5,49 @@
...
+                <ul>
+                    <li> Our support email: support@awesomebook.com</li>
+                    <li>You can call us: 0532214239</li>
+                    <li>Our address is: New Human Colony, Mars</li>
+                </ul>
+            </div>
+        </section>
+    </main>
+
+    <footer>Copyright@2023</footer>
```
Comment (HAMMAS-SALEEM) [inline index.html:54]: - [x] Great work on implementing the requirements of this project 👍 but this project has a crucial requirement of pair programming 🤔 I cannot see any commit from one of your team members @bertinkambale 😨 Kindly ask him to contribute to your project to follow this crucial requirement 👍

**Review by HAMMAS-SALEEM — COMMENTED** [review]
# Invalid Submission :red_circle:

invalid-submission

## Hey @AbhishekkM05, @Bennyjoez, @bertinkambale :wave:

## Highlights :1st_place_medal:

✔️ Descriptive PR Title
✔️ Descriptive PR Summary
✔️ Following GitHub Flow
✔️ Following some requirements

Good job on implementing some of the requirements but I cannot proceed review of your pull request because there are some issues that you still need to work on to go to the next project but you are almost there! :blush:

# Optional

_All the changes with optional tags are not crucial enough to prevent you from getting the approval but we highly recommend to students that they should take these changes into consideration._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

---------------------------------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (Bennyjoez) [reply index.html:54]: @HAMMAS-SALEEM Thank you 👍🏾 for the review. The one **@bertinkambale** unexpectedly quit the coding partners group. We got a message from the Otto bot on the slack channel that Bertin will no longer be with us. 😞

**Review by rloterh — COMMENTED** [review]
Hi Team, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

Kindly ensure that your ReadMe is professionally documented, **remove Live demo link** if app is not yet deployed.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 2]
```diff
@@ -5,19 +5,49 @@
...
-        <button type="submit">Add</button>
-    </form>
+    <header>
+        <nav>Awesome book
+            <ul id="navbar">
+                <li class="nav-btn" data-id="book-list-section">Book-list</li> 
+                <li class="nav-btn" data-id="form">Add new</a></li> 
+                <li class="nav-btn" data-id="Contact">Contact</li> 
+            </ul>
+        </nav>
```
Comment (mckent05) [inline index.html:19]: **OPTIONAL** - You have done well so far, but there are differences between your project and the wireframe, your website is missing the clock/time that is visible just below the navbar. Please fix this 🙏 

Your project:
everv

**Review by mckent05 — APPROVED** [review]
Hi @AbhishekkM05, 

# STATUS: APPROVED ✔️ ✔️ 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations and well done! 🎉 
approved


## Highlights
- All linter checks are passing ✔️ ✔️ 
- Your PR is professional 👏 👏 
- Your code is clean and easy to read, well-done 👍 👍 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (Bennyjoez) [reply index.html:19]: Optional issue fixed in recent commits. 😄

**Pull request merged**